### DISCUSSION TITLE
Request/Suggestion - Make the helicopters flyable by the player

### DISCUSSION THREAD

**Discussion by Gu5-Fring** [TEHE-Studios/ExpandedHelicopterEvents#224] [Q&A]
I want to suggest adding the function to fly the helicopters added by the mod or maybe add compatibility with the [UH-60 Black Hawk Mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2799925193) or even adding the functionality of the UH-60 Black Hawk Mod to the Expanded Helicopter Events Mod.

**Comment by sharkster91** [comment]
No.
